source/reference/minio-mc/mc-support-perf.rst

```diff
@@ -0,0 +1,224 @@
+===================
+``mc support perf``
+===================
+
+.. default-domain:: minio
+
+.. contents:: Table of Contents
+   :local:
+   :depth: 1
+
+.. mc:: mc support perf
+
+Description
+-----------
+
+Use the :mc-cmd:`mc support perf` command to analyze the performance of the S3 read/write, network I/O, or drive read/write.
+
+The resulting test can provide general guidance of deployment performance under S3 ``GET`` and ``PUT`` requests and identify any potential bottlenecks.
+
+For more complete performance testing, consider using a combination of load-testing using your staging application environments and the MinIO `WARP <https://github.com/minio/warp>`_ S3 benchmarking tool.
+   
+:mc-cmd:`mc support perf` has three subcommands
+
+#. :mc-cmd:`~mc support perf drive`
+
+   Measure the speed of a drive in a cluster.
+
+#. :mc-cmd:`~mc support perf object`
+      
+   Measure the speed of reading and writing objects in a cluster.
+
+#. :mc-cmd:`~mc support perf net`
+
+   Measure the network throughput of all nodes.
+
+Examples
+--------
+
+Measure Speed of an Object
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Measure the performance of S3 read/write of an object on the alias ``minio1``.
+MinIO autotunes concurrency to obtain maximum throughput and IOPS (Input/Output Per Second).
+
+.. code-block:: shell
+   :class: copyable
+ 
+   mc support perf object minio1
+
+Measure Speed of an Object of a Specific Size for a Specific Duration
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Run object the S3 read/write performance of an object for 20 seconds with object size of 128MiB on alias ``minio1``.
+MinIO autotunes concurrency to obtain maximum throughput.
+
+.. code-block:: shell
+   :class: copyable
+
+   mc support perf object minio1 --duration 20s --size 128MiB
+
+
+Test Speed of All Drives on All Nodes with Default Specifications
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Run drive read/write performance measurements on all drive on all nodes for a cluster with alias ``minio1``.
+The command does not specify the blocksize, so the default of 4MiB is used.
+
+.. code-block:: shell
+   :class: copyable
+ 
+   mc support callhome get minio1
```

Comment:
I think this is the wrong command - should be mc support drive?

Reply:
😬